src/lib/coerce.js

```diff
@@ -114,8 +116,15 @@ exports.valObjectMeta = {
             'are coerced to the `dflt`.'
         ].join(' '),
         requiredOpts: [],
-        otherOpts: ['dflt', 'min', 'max', 'arrayOk'],
+        otherOpts: ['dflt', 'min', 'max', 'arrayOk', 'extras'],
         coerceFunction: function(v, propOut, dflt, opts) {
+            if((opts.extras || []).indexOf(v) !== -1) {
+                propOut.set(v);
+                return;
+            }
+
+            if(isTypedArraySpec(v)) v = decodeTypedArraySpec(v);
```

Comment:
Fix for handling base64 encoded arrays.
cc: https://github.com/plotly/plotly.js/pull/5230 and https://github.com/plotly/plotly.py/pull/4470